airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml

```diff
@@ -11,7 +11,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 137ece28-5434-455c-8f34-69dc3782f451
-  dockerImageTag: 2.1.0
+  dockerImageTag: 2.1.1
```

Comment:
please remove `dockerImageTag: 2.0.0` as issue is fixed now

Reply:
Removed